packages/gitbook/src/components/Header/CompactHeader.tsx

```diff
@@ -20,15 +20,12 @@ import { SearchButton } from '../Search';
  */
 export function CompactHeader(props: {
     space: Space;
-    parent: Site | Collection | null;
+    site: Site | null;
```

Comment:
How is it possible to have a header without a `site` ?

Reply:
`Header` is rendered by `SpaceLayout` which can currently render without a site (ex - for  pdf export )